### PR TITLE
Add harvester cloud provider chart

### PR DESCRIPTION
**Pull request by yaocw2020**
issue: https://github.com/harvester/harvester/issues/1019

### CODE REVIEW

**Comment by yaocw2020** [comment]
> @yaocw2020 can u rebase those charts so it will run against the new setup CI, thanks

Done

**Comment by yaocw2020** [comment]
It should be merged after the version 0.1.1 release.

File: charts/harvester-cloud-provider/values.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicasCount: 2
+
+image:
+  repository: rancher/harvester-cloud-provider
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
```
Comment (guangbochen) [inline charts/harvester-cloud-provider/values.yaml:11]: set default to `master-head` as this is a master version

**Review by guangbochen — COMMENTED** [review]
Will need to fix the CI

**Comment by yaocw2020** [comment]
> Will need to fix the CI

Fix!!

**Review by guangbochen — APPROVED** [review]
LGTM

**Pull request merged**